REST/PowerShell/Variables/ChangeVariableValueinVariableSet.ps1

```diff
@@ -0,0 +1,92 @@
+#Allows HTTPS connections
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+$VarSetName = "YourVariableSetName"           #The exact name of the Variable Set you are trying to change a variable within. Not needed if you know ID
+$VarSetID = ""                                #Leave blank if you dont know your variable set ID
+
+$VariableName = "YourVariableName"            #The name of the variable you are trying to change the value of
+$VariableValue = "YourValue"                  #New value of Variable
+$VariableEnvName = "YourEnvironment"          #Scope of a variable to a environment, leave blank to scope to all
+$OverrideMultipleEnv = $False                 #True will replace Value for environments will multiple scopes i.e. if a environment is part of 2 scopes it will replace both 
+                                              #False will cause an error if the environment is shared with more than value or if multiple scopes have the environment
```

Comment:
These two comments are also confusing. It's *variables* with multiple scopes, not **environments**

Reply:
Thanks for the feedback. I will add the support for space and clean those comments up.